formidable/models.py

```diff
@@ -53,10 +53,14 @@ def from_json(definition_schema, **kwargs):
         from formidable.serializers import FormidableSerializer
 
         serializer = FormidableSerializer(data=definition_schema)
-        if serializer.is_valid():
-            return serializer.save(**kwargs)
+        if not serializer.is_valid():
```

Comment:
I'm not sure about this change... why would it happen, and also, why would it happen on this PR?